docs/overview/learning-path-transition.md

```diff
@@ -1,26 +1,28 @@
 ---
-title: VSTO add-in developer's guide
+title: VSTO add-in developer's guide to Office Web Add-ins
 description:  A recommended path for experienced VSTO add-in developers to learning resources for Office Web Add-ins.
-ms.date: 10/14/2020
+ms.date: 02/07/2023
 ms.custom: scenarios:getting-started
 ms.localizationpriority: high
 ---
 
-# VSTO add-in developer's guide
+# VSTO add-in developer's guide to Office Web Add-ins
 
 So, you've made some VSTO add-ins for Office applications that run on Windows and now you're exploring the new way of extending Office that will run on Windows, Mac, and the web browser version of the Office suite: Office Web Add-ins.
 
+[!INCLUDE [new-outlook-vsto-com-support](../includes/new-outlook-vsto-com-support.md)]
+
 Your understanding of the object models for the Excel, Word, and the other Office applications will be a huge help because the object models in Office Web Add-ins follow similar patterns. But there are going to be some challenges:
 
-- You will be working with a different language (either JavaScript or TypeScript) instead of C# or Visual Basic .NET. (There is also a way, described below, to reuse some of your existing code in a web add-in.)
+- You'll be working with a different language (either JavaScript or TypeScript) instead of C# or Visual Basic .NET. (There is also a way, described below, to reuse some of your existing code in a web add-in.)
```

Comment:
```suggestion
- You'll be working with a different language (either JavaScript or TypeScript) instead of C# or Visual Basic .NET. (There is also a way, described later, to reuse some of your existing code in a web add-in.)
```